pkg/gardenlet/controller/backupentry/backup_entry.go

```diff
@@ -115,26 +129,25 @@ func (c *Controller) Run(ctx context.Context, workers, migrationWorkers int) {
 	var waitGroup sync.WaitGroup
 
 	if !cache.WaitForCacheSync(ctx.Done(), c.backupEntryInformer.HasSynced, c.seedInformer.HasSynced) {
-		logger.Logger.Fatal("Timed out waiting for caches to sync")
+		c.log.Error(wait.ErrWaitTimeout, "Timed out waiting for caches to sync")
```

Comment:
logrus.Fatal() panics while logr.Error() is not. Is this changing behaviour desired?

Reply:
Not sure why there was this `Fatal()` call before, but it doesn't look good (in all other controllers we just error-log this)